### ISSUE TITLE
Support for dart environment definitions

### ISSUE THREAD

**Issue by andynewman10** [lamnhan066/isolate_manager#26]
Currently when generating the isolate files, the following command is run
```
dart run isolate_manager:generate
```
Options can be specified on the command line (obfuscation, debug...), however they lack the ability to add dart definitions using `-D`:
```
dart compile js ... -Dfoo=bar
```
Environment variables can be accessed through the code using, eg. `bool.fromEnvironment()`.

`-D` definitions are important, as they may include specific values for debug and release (production) builds.

Adding a `-D` or `--dart-define` parameter (and ensuring multiple defines can be added) would be very valuable.

**Comment by lamnhan066** [comment]
Hi @andynewman10 

I have created a PR for your issue. What do you think about my idea?

**Comment by andynewman10** [comment]
Looks good to me!

**Comment by andynewman10** [comment]
@lamnhan066 is it me or temporary `lib/.IsolateManagerWorker.* `files are no longer getting cleaned out once the generation process is finished?

**Comment by lamnhan066** [comment]
@andynewman10 please open a new issue with the command that you're running. That will make it easier for me to track and resolve the issue. Thank you.
